projects/gallery/src/lib/components/viewer/viewer.component.ts

```diff
@@ -492,7 +497,11 @@ export class ViewerComponent implements OnChanges, OnInit, OnDestroy {
   }
 
   private shift(x: number) {
-    this.itemListRef.nativeElement.style.transform = `translate3d(${-(this._listX = x)}px, 0, 0)`;
+    let multiplier = -1;
+    if (this.isRtl) {
+      multiplier = 1;
+    }
+    this.itemListRef.nativeElement.style.transform = `translate3d(${multiplier * (this._listX = x)}px, 0, 0)`;
```

Comment:
It would be nice to write a unit test for this, in the same spirit as there are already tests for the "normal" shifts. Can you manage?

Reply:
Sorry, I have attempted to write a test but it kept failing I am not sure what was wrong with the test - I duplicated one of the shifting tests and set up the component isRtl to true and expected a positive result from translateX function (instead of negative), but it kept on failing.

Reply:
Hi, I think the problem was in the direction that you slid. No worries, I wrote the tests.